lib/fly.js

```diff
@@ -81,16 +81,28 @@ module.exports = Fly
  * Compose a new, yeildable sequence.
  * Resets instance's glob, filters, and writer.
  *
- * @param  {String|Array} globs The glob selection
+ * @param  {String|Array} ...globs The glob selection
+ * @param {Object} options If the last argument is an object it's used as options to pass `node-glob`
  * @return                      The current Fly instance.
  */
-Fly.prototype.source = function (globs) {
-	globs = arrify(globs)
+Fly.prototype.source = function () {
+	var globs = flatten([].slice.call(arguments))
+	var options = globs.pop()
```

Comment:
If the last option **isn't** an option object then you're losing a `glob` entry for the `map` function below.